src/quantem/tomography/tomography_logger.py

```diff
@@ -0,0 +1,70 @@
+import matplotlib.pyplot as plt
+
+from quantem.core.ml.logger import LoggerBase
+from quantem.tomography.object_models import ObjectModelType
+from quantem.tomography.tomography_dataset import TomographyDataset
+
+
+class TomoLogger(LoggerBase):
```

Comment:
I'd suggest naming conventions like:
LoggerTomography

I prefer the parent name coming first if possible to make finding things easier when looking through attributes.